client/src/modules/invoices/templates/grid/code.tmpl.html

```diff
@@ -14,7 +14,7 @@
     <span
       class="text-info"
       ng-show="row.entity._hasPriceList"
-      uib-popover="{{ 'PATIENT_INVOICE.BASE_PRICE_ALTERED' | translate }}"
+      translate-attr="{ 'uib-popover': 'PATIENT_INVOICE.BASE_PRICE_ALTERED' }"
```

Comment:
I don't think this will work, unfortunately,  `uib-popover` is a directive, not an attribute.  Can you confirm that it works?  If not, we'll have to leave it the way it was.